.changeset/seven-rings-yell.md

```diff
@@ -0,0 +1,12 @@
+---
+"thirdweb": minor
```

Comment:
let's also mention `login` and `logout` commands in this changeset

Reply:
Updated